runtime/doc/vim_diff.txt

```diff
@@ -31,6 +31,7 @@ these differences.
 - 'backspace' defaults to "indent,eol,start"
 - 'encoding' defaults to "utf-8"
 - 'nocompatible' is always set
+- 'nrformats' doesn't include "octal" by default
```

Comment:
This should probably follow the scheme of `'option' defaults "string"` used in this paragraph.